docs/workflow/building/coreclr/android.md

```diff
@@ -1,15 +1,59 @@
-Cross Compilation for Android on Linux
-======================================
+# Experimental support of CoreCLR on Android
+
+This documentation outlines experimental support of CoreCLR on Android.
+It includes instructions on how to:
+- [Build CoreCLR for Android](./android.md#building-coreclr-for-android)
+- [Build and run a sample Android application with CoreCLR](./android.md#building-and-running-helloandroid-sample-app)
+- [Debug the sample app and the runtime](./android.md#debugging-the-sample-app)
+
+## Prerequisite
+
+Download and install `Android Studio` and the following:
+  - Android SDK (minimum supported API level is 21)
+  - Android NDK r27
+
+## Building CoreCLR for Android
+
+Supported host systems for building CoreCLR for Android:
+- [MacOS](./android.md#macos) ✔
+- [Linux](./android.md#linux) ✔
+- [Windows](./android.md#windows) ❌ (only through WSL)
+
+Supported target architectures:
+- x86 ❌
+- x64 ✔
+- arm ❌
+- arm64 ✔
+
+### MacOS
+
+#### Requirements
+
+Set the following environment variables:
+  - ANDROID_SDK_ROOT=`<full-path-to-android-sdk>` (example:`ANDROID_SDK_ROOT=/Users/<user>/Library/Android/sdk`)
+  - ANDROID_NDK_ROOT=`<full-path-to-android-ndk>` (example:`ANDROID_NDK_ROOT=/Users/<user>/Library/Android/sdk/ndk/27.2.12479018`)
+
+#### Building the runtime, libraries and tools
+
+To build CoreCLR runtime packages, libraries and tools, run the following command from `<repo-root>`:
+
+```
+./build.sh clr.runtime+clr.alljits+clr.corelib+clr.nativecorelib+clr.tools+clr.packages+libs+packs -os android -arch <x64|arm64> -c <Debug|Release> -cross -bl
+```
+
+NOTE: The runtime packages will be located at: `<repo-root>/artifacts/packages/<configuration>/Shipping/`
+
+### Linux
+
+TODO: Bellow are the old notes which we need to reevaluate
```

Comment:
@janvorli could you please check if the Linux notes are still relevant?
or if not, could you please update them?

Reply:
Let me update it, some of the commands are incorrect.